src/pybind/mgr/dashboard/module.py

```diff
@@ -14,9 +14,16 @@
 import tempfile
 import threading
 import time
-from typing import Optional
+from typing import TYPE_CHECKING, Optional
 
-from mgr_module import CLIWriteCommand, MgrModule, MgrStandbyModule, Option, _get_localized_key
+if TYPE_CHECKING:
+    if sys.version_info >= (3, 8):
+        from typing import Literal
+    else:
+        from typing_extensions import Literal
```

Comment:
No need to change, but just as a reference, [this is how other libraries do that](https://github.com/agronholm/typeguard/blob/master/src/typeguard/__init__.py#L26-L32).

Reply:
Thanks @alfonsomthd for addressing my suggestions! :D

Reply:
> No need to change, but just as a reference, [this is how other libraries do that](https://github.com/agronholm/typeguard/blob/master/src/typeguard/__init__.py#L26-L32).

I just applied the same approach as [here](https://github.com/ceph/ceph/pull/36146/commits/f002c6ce40338c08dbc91e08682365e07662f135) :grin: